third-party/realdds/include/realdds/dds-participant.h

```diff
@@ -71,6 +75,12 @@ class dds_participant
         qos( std::string const & participant_name );
     };
 
+    // Return the QoS (so user won't have to actually know about the DomainParticipant)
+    eprosima::fastdds::dds::DomainParticipantQos const & get_qos() const;
```

Comment:
Who is using this? Should be public?

Reply:
Just saw internal usage, if only internal then no need to overload the public API. You can leave public if you think it's necessary.